<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRCodeSequence</class>
 <widget class="QWidget" name="QRCodeSequence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="prevButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>60</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>60</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
<property name="autoDefault">
<bool>false</bool>
</property>
  </widget>
  <widget class="QLabel" name="pageLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>1/10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="qrCode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
